app/models/sales_log.rb

```diff
@@ -0,0 +1,72 @@
+class SalesLogValidator < ActiveModel::Validator
+  def validate(record); end
+end
+
+class SalesLog < Log
+  has_paper_trail
+
+  validates_with SalesLogValidator
+  before_save :update_status!
+
+  scope :filter_by_organisation, ->(org, _user = nil) { where(owning_organisation: org).or(where(managing_organisation: org)) }
+
+  STATUS = { "not_started" => 0, "in_progress" => 1, "completed" => 2 }.freeze
```

Comment:
Should this stuff be in the parent model? Sales and Lettings logs will have the same statuses right?

Reply:
I would assume so! Will move it across and can always reintroduce it here if that doesn't turn out to be true 😄

Reply:
https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/858/commits/e27901f32d0416fc7dd20db0c58502ea0672c75b